src/apic-extension/azext_apic_extension/aaz/latest/apic/api/_create.py

```diff
@@ -324,7 +327,7 @@ def content(self):
                 typ=AAZObjectType,
                 typ_kwargs={"flags": {"required": True, "client_flatten": True}}
             )
-            _builder.set_prop("properties", AAZObjectType, typ_kwargs={"flags": {"client_flatten": True}})
+            _builder.set_prop("properties", AAZObjectType, ".", typ_kwargs={"flags": {"required": True, "client_flatten": True}})
```

Comment:
Confirmed with Azure CLI team, the change is expected.

Background:
typespec has a bug that treats some required parameter as optional. I patch the spec in the way suggested by Azure CLI team and causes this change. CLI users will not be aware of this change.